_specs/ecip-1000.md

```diff
@@ -73,10 +73,10 @@ A good reason to transfer ownership is because the original author no longer has
 The current ECIP editors are:
 
 * Wei Tang (@sorpaas)
-* Isaac Ardis (@meowsbits)
 * Cody Burns (@realcodywburns)
-* Afri Schoedon (@soc1c)
 * Yaz Khoury (@YazzyYaz)
+* @soc1c (Talha Cross, Pseudonymous)
+* @meowsbits (Mr. Meows D. Bits, Pseudonymous)
```

Comment:
I would prefer not to disclose whether or not this my pen name or my fiat name. I prefer a certain air of mystery... :rabbit: :tophat: 

```suggestion
* @meowsbits (Mr. Meows D. Bits)
```

Reply:
This is ECIP process. I don't think it's the right place for mystery.

Reply:
I'm going to appeal this later, but that's fine with me for now.

The important thing is get the change related to @soc1c in ASAP. Please.